react/src/App.js

```diff
@@ -62,7 +60,7 @@ function App() {
       setIsAuthenticated(true);
     } catch (e) {
       if (e !== "No current user") {
-        alert(e);
+        console.error(e);
```

Comment:
ditto remove console logging. if need be, pls use some proper notification modal or Snackbar to feedback user.